src/service/fightData.ts

```diff
@@ -0,0 +1,216 @@
+import { randomValue, type Range } from "@/service/random.js";
+
+export const fightTemplates: { [name: string]: Equipable } = {
```

Comment:
Hier geht auch ein Record:
```suggestion
export const fightTemplates: Record<string, Equipable> = {
```